README.md

```diff
@@ -148,6 +149,45 @@ const config = cfg();
 //     value at "b" cannot be undefined
 ```
 
+### `assembleFrom` Example
+
+**NOTE**: This only affects URL values when `format` is set to `url`.
+
+Sometimes you can't provide a connection string and have to build it up by
+parts, specified as individual environment variables. For those cases, you can
+provide the `assembleFrom` configuration option, which accepts a map like this:
+
+```json
+{
+  "url-in-parts": {
+    "desc": "a url in parts",
+    "env": "URL",
+    "format": "url",
+    "assembleFrom": {
+      "host": "URL_HOST",
```

Comment:
If the consumer uses `DB_HOST` as config name, does this convert it automatically? Or have to be identical to `host`?

Reply:
The map just maps the expected internal value `host` to the user-provided value `URL_HOST`/`DB_HOST`/`WHATEVER_HOST`.

The keys in this map cannot change; the values are whatever you need them to be.